### PR TITLE
[Inference]Add dropout and depthwise_conv2d

### PR DESCRIPTION
**Pull request by YuanRisheng**
### PR Category

Inference

### PR Types

Others

### Description

card-71500
本PR主要做了如下工作：
1，添加dropout和depthwise_conv2d的trt converter

### CODE REVIEW

File: python/paddle/tensorrt/impls/common.py [hunk 1]
```diff
@@ -12,10 +12,40 @@
...
+    dropout_prob = p_defining_op.attrs()["value"]
+    downgrade_in_infer = paddle_op.attrs().get("mode")
+
+    if downgrade_in_infer == "upscale_in_train":
+        shuffle_layer = network.add_shuffle(input_x)
+        return shuffle_layer.get_output(0)
+
+    weight_data = np.array([1 - dropout_prob]).astype("float32")
+    scale_weights = trt.Weights(weight_data)
+    shift_weights = trt.Weights(np.array([0]).astype("float32"))
```
Comment (lizexu123) [inline python/paddle/tensorrt/impls/common.py:37]: 这边直接写float32，以后会不会是float16类型

Comment (YuanRisheng) [reply python/paddle/tensorrt/impls/common.py:37]: 这个应该问题不大，老IR的converter就是固定的数据类型和固定数值

Comment (lizexu123) [reply python/paddle/tensorrt/impls/common.py:37]: 哦哦

**Review by lizexu123 — APPROVED** [review]

File: python/paddle/tensorrt/impls/common.py [hunk 2]
```diff
@@ -31,7 +61,8 @@ def bilinear_interp_converter(network, paddle_op, inputs):
     out_d = paddle_op.attrs().get("out_d")
     scale_attr = paddle_op.attrs().get("scale")
 
-    trt_major, trt_minor, trt_patch = trt.__version__.split(".")
+    trt_major = get_trt_version_list()[0]
+    trt_minor = get_trt_version_list()[1]
```
Comment (vivienfanghuagood) [inline python/paddle/tensorrt/impls/common.py:65]: trt Version的特化代码，是否考虑“模板化”一点？

**Review by vivienfanghuagood — APPROVED** [review]

File: python/paddle/tensorrt/impls/common.py (hunk 2 above)
Comment (YuanRisheng) [reply python/paddle/tensorrt/impls/common.py:65]: 下个pr修改

**Review by yuanlehome — APPROVED** [review]

File: test/tensorrt/test_converter_conv.py [hunk 3]
```diff
@@ -37,5 +37,22 @@ def test_trt_result(self):
         self.check_trt_result()
 
 
+def depthwise_conv2d_wrapper(x):
+    conv = paddle.nn.Conv2D(2, 2, (3, 3), groups=2)
```
Comment (lizexu123) [inline test/tensorrt/test_converter_conv.py:41]: 刚发现你这里测的是Conv2d
